src/main/java/frc/utils/ChargeStationStates.java

```diff
@@ -0,0 +1,127 @@
+package frc.utils;
+
+import edu.wpi.first.wpilibj.BuiltInAccelerometer;
+
+public class ChargeStationStates {
+    private BuiltInAccelerometer mRioAccel;
+    private States state;
+    private int debounceCount;
+    private double robotSpeedSlow;
+    private double robotSpeedFast;
+    private double onChargeStationDegree;
+    private double levelDegree;
+    private double debounceTime;
+
+    public ChargeStationStates() {
+        mRioAccel = new BuiltInAccelerometer();
+        state = States.APPROACH;
+        debounceCount = 0;
+
+        // Speed the robot drived while scoring/approaching station, default = 0.4
+        robotSpeedFast = 0.4;
+
+        // Speed the robot drives while balancing itself on the charge station.
+        // Should be roughly half the fast speed, to make the robot more accurate,
+        // default = 0.2
+        robotSpeedSlow = 0.2;
+
+        // Angle where the robot knows it is on the charge station, default = 13.0
+        onChargeStationDegree = 13.0;
+
+        // Angle where the robot can assume it is level on the charging station
+        // Used for exiting the drive forward sequence as well as for auto balancing,
+        // default = 6.0
+        levelDegree = 4.0;
```

Comment:
Is this the official value from the game manual?

Reply:
Empirically tested